_posts/2024-03-11-release-0_5_0.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title:  "Kroxylicious release 0.5.0"
+date:   2024-03-10 00:00:00 +0000
+author: "Robert Young"
+author_url: "https://www.github.com/robobario"
+categories:  [releases, kroxylicious, record-encryption]
+---
+
+The Kroxylicious project is very pleased to announce the release of [Kroxylicious 0.5.0](https://github.com/kroxylicious/kroxylicious/releases/tag/v0.5.0).
+
+### Record Encryption
+
+This release was focused on refining [Record Encryption](/use-cases) to be ready for experimentation in secure environments, adding:
+* Enable users to configure Hashicorp Vault TLS, providing custom keystore and truststore
+* Enable users to supply the Hashicorp Vault token via file rather than having to inject it into the proxy configuration
+* Include Record Encryption filter (and other Kroxylicious project supported filters) in the binary distribution
+* Release a Docker image `quay.io/kroxylicious/kroxylicious` also containing the supported filters
+* Internal refactoring to control the usage of key material, preparing to support alternate Ciphers and configurable Additional Authenticated Data
+
+The protocol for immutable encrypted data written to the broker is now aligned with our initial design and we guarantee data encrypted with version 0.5.0 
+of the Filter will be decryptable forever (assuming the keys stored in the KMS are available forever for decryption).
+
+> **_NOTE:_**: Until now we have used the term Envelope Encryption when describing the Filter. Going forward we will use the name Record Encryption 
+to better describe the granularity that encryption is happening at. Encryption is done per [record](https://kafka.apache.org/documentation/#record).
+Envelope Encryption is how we encrypt those records. Record Encryption is how we implement Encryption-At-Rest.
```

Comment:
This could possibly go in a [bootstrap alert element](https://getbootstrap.com/docs/5.3/components/alerts/), we have a pre-made one you can include at [`_includes/bs-alert.html`](https://github.com/kroxylicious/kroxylicious.github.io/blob/main/_includes/bs-alert.html) (there are examples of it being used in [`_quickstarts/deploy.markdown`](https://github.com/kroxylicious/kroxylicious.github.io/blob/main/_quickstarts/deploy.markdown?plain=1#L24)). I think it looks fine as it is, but using the `bs-alert` element might make it a bit more uniform with the quickstarts.

For example:
```suggestion
{% capture record_encryption_note %}
Until now we have used the term Envelope Encryption when describing the Filter. Going forward we will use the name Record Encryption 
to better describe the granularity that encryption is happening at. Encryption is done per [record](https://kafka.apache.org/documentation/#record).
Envelope Encryption is how we encrypt those records. Record Encryption is how we implement Encryption-At-Rest.
{% endcapture %}

{% include bs-alert.html type="primary" icon="info-circle-fill" content=record_encryption_note %}
```

Which looks a bit like this:
![image](https://github.com/kroxylicious/kroxylicious.github.io/assets/15976368/7cb218d7-b6bd-4661-829a-69137e5bd108)

Reply:
Have updated to use the banner